### PR TITLE
[DTM][1018] Content for DTM v1018

### PR DESCRIPTION
**Pull request by lra-sag**
Documentation content for
1. DTM app name change from Digital Twin Manager to Digital twin manager 
2. Import and Export of asset models and asset properties
3. Relocation of assets between hierarchies
4. Updated screenshots with new app name

### CODE REVIEW

**Review by pavanshivajirao — APPROVED** [review]

File: content/dtm/asset-types-bundle/asset-types.md [hunk 1]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+
+### To export asset models
+
+To export asset models to add in another tenant, follow the steps below:
+
+1. In the **Asset models** page, click **Export asset models**. 
+2. In the resulting dialog, you see the list of asset models. Initially, all the asset models are collapsed. Click the expand icon to the left of each asset model to view the next level of child asset models.
+3. Select one or more desired asset models.
+4. Click **Export** to export the selection.
+5. The dialog closes and a JSON file named 'Export-asset-models.json' downloads.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:214]:
```suggestion
5. The dialog window closes and a JSON file named 'Export-asset-models.json' downloads.
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 2]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+### To import asset models
+
+To import asset models, follow the steps below:
+
+1. In the **Asset models** page, click **Import asset models**.
+2. Upload the JSON file in the drop file area of the resulting dialog. The JSON file is the file containing exported asset models from a DTM application on another tenant. 
+3. Click **Next** to preview the asset models. Initially, all the asset models are collapsed. Click the expand icon to the left of each asset model to view the next level of child asset models.
+3. Click **Import** to import the asset models.
+4. Or, click **Back** to go to previous step. Hover over the file name and click the delete icon to remove the current file and upload a different one.
+4. View newly imported asset models in the **Asset models** page.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:239]:
```suggestion
4. The newly imported asset models are listed on the **Asset models** page.
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 3]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+
+![Import asset models](/images/dtm/asset-type/dtm-asset-type-import.png)
+{{< c8y-admon-info>}}
+When you upload the JSON file, it undergoes validation. If the validation fails, you see a corresponding message in the dialog. Click **Download log file** to view the errors.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:243]:
```suggestion
When you upload the JSON file, it undergoes a validation process. If the validation fails, you see a corresponding message in the resulting dialog box. Click **Download log file** to view the errors.
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 4]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+Review the errors that are reported, and attempt the import again after addressing the errors.
+
+You can import a maximum of 250 asset models and 500 associated properties at one time.
+{{< /c8y-admon-info>}}
+
+### To import asset model samples
+
+When you navigate to **Asset model samples** page following **Configuration > Asset models > Asset model samples**, you see a list of samples with name, description and tags associated with it.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:251]:
```suggestion
When you navigate to the **Asset model samples** page following **Configuration > Asset models > Asset model samples**, you see a list of samples with name, description and tags associated with it. 
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 5]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+Review the errors that are reported, and attempt the import again after addressing the errors.
+
+You can import a maximum of 250 asset models and 500 associated properties at one time.
+{{< /c8y-admon-info>}}
+
+### To import asset model samples
+
+When you navigate to **Asset model samples** page following **Configuration > Asset models > Asset model samples**, you see a list of samples with name, description and tags associated with it. 
+
+To view the sample, click the menu icon to the right of sample name and click the View option.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:253]:
```suggestion
To view the sample, click the menu icon to the right of sample name and click **View**. 
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 6]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+
+### To import asset model samples
+
+When you navigate to **Asset model samples** page following **Configuration > Asset models > Asset model samples**, you see a list of samples with name, description and tags associated with it. 
+
+To view the sample, click the menu icon to the right of sample name and click the View option. 
+In the resulting dialog, you see a list of asset models. Initially, the asset models in the list are collapsed. Click the expand icon to the left of each asset model to view the complete hierarchy.
+Click **Import** to import the asset models in the sample.
+
+Alternatively, to import the asset models in the sample without a preview, click the menu icon to the right of sample name and click the Convert to asset model option.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:257]:
```suggestion
Alternatively, to import the asset models in the sample without a preview, click the menu icon to the right of sample name and then click **Convert to asset model**.
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 7]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+
+{{< c8y-admon-info>}}
+When you choose View or Convert to asset model option, the asset model sample undergoes validation. If the validation fails, you see a corresponding message in the dialog. Click **Download log file** to view the errors.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:260]:
```suggestion
When you choose **View** or **Convert to asset model**, the asset model sample undergoes a validation process. If the validation fails, you see a corresponding message in the resulting dialog box. Click **Download log file** to view the errors.
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 8]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+Review the errors that are reported, and attempt the action again after addressing the errors.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:261]:
```suggestion
Review the errors that are reported, resolve them and attempt the action again.
```

File: content/dtm/asset-types-bundle/asset-types.md [hunk 9]
```diff
@@ -197,3 +202,61 @@ To sort columns, hover over the respective column header and click the sort icon
...
+2. Upload the JSON file in the drop file area of the resulting dialog. The JSON file is the file containing exported asset models from a DTM application on another tenant. 
+3. Click **Next** to preview the asset models. Initially, all the asset models are collapsed. Click the expand icon to the left of each asset model to view the next level of child asset models.
+3. Click **Import** to import the asset models.
+4. Or, click **Back** to go to previous step. Hover over the file name and click the delete icon to remove the current file and upload a different one.
+4. View newly imported asset models in the **Asset models** page.
+
+![Import asset models](/images/dtm/asset-type/dtm-asset-type-import.png)
+{{< c8y-admon-info>}}
+When you upload the JSON file, it undergoes validation. If the validation fails, you see a corresponding message in the dialog. Click **Download log file** to view the errors.
+Review the errors that are reported, and attempt the import again after addressing the errors.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/asset-types.md:244]:
```suggestion
Review the errors that are reported, resolve them, and attempt the import again.
```

File: content/dtm/asset-types-bundle/property-library.md [hunk 10]
```diff
@@ -294,4 +296,43 @@ You see the preview of data model on the right side of properties section.
...
 
-![Complex asset property](/images/dtm/custom-property/dtm-property-library-complex-property.png)
\ No newline at end of file
+![Complex asset property](/images/dtm/custom-property/dtm-property-library-complex-property.png)
+
+### To export asset properties
+
+Follow the steps below:
+
+1. In the **Asset properties** page, click **Export asset properties**. In the resulting dialog, you see the list of asset properties along with their description and type.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/property-library.md:305]:
```suggestion
1. On the **Asset properties** page, click **Export asset properties**. In the resulting dialog box, you see the list of asset properties along with their description and type.
```

File: content/dtm/asset-types-bundle/property-library.md [hunk 11]
```diff
@@ -294,4 +296,43 @@ You see the preview of data model on the right side of properties section.
...
+You can export a maximum of 500 asset properties at a time.
+{{< /c8y-admon-info>}}
+
+![Export asset properties](/images/dtm/custom-property/dtm-property-library-export.png)
+
+### To import asset properties
+
+To import asset properties into the DTM application, follow the steps below:
+
+1. In the **Asset properties** page, click **Import asset properties**.
```
Comment (MWindel) [inline content/dtm/asset-types-bundle/property-library.md:321]:
```suggestion
1. On the **Asset properties** page, click **Import asset properties**.
```

File: content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md [hunk 12]
```diff
@@ -45,15 +45,15 @@ Follow the steps below:
 
 1. Navigate to the **Assets** page and click **Import assets**.
 
-2. The **Import assets** dialog has a dropdown **Choose asset model**. If you have added the asset models, all the root asset models are listed in the dropdown.
+2. The **Import assets** dialog has a dropdown **Select asset model**. If you have added the asset models, all the root asset models are listed in the dropdown.
```
[the remark this answers is not in the document]
Comment (lra-sag) [reply content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md:48]: In the latest version of 1018 of DTM, this label is changed to 'Select asset model'
Here is the screenshot. What is the right way to identify a dropdown - using its label or place holder text, in this case Select asset model and Choose asset model respectively.
image

[the remark this answers is not in the document]
Comment (lra-sag) [reply content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md:48]: @MWindel, thank you for the clarification. We will make the changes accordingly.

**Review by MWindel — APPROVED** [review]

**Pull request merged**